client/src/components/App.jsx

```diff
@@ -1,24 +1,88 @@
+/* eslint-disable no-console */
 import React from 'react';
-import ReviewOverview from './ReviewOverview.jsx';
-import MostHelpfulReviews from './MostHelpfulReviews.jsx';
-import SortAndProgress from './SortAndProgress.jsx';
-import IndividualReview from './IndividualReview.jsx';
-import NextPageAndProgress from './NextPageAndProgress.jsx';
+import axios from 'axios';
 
-const App = () => (
-  <div>
-    <div className="inline-block">
-      <span>Reviews&nbsp;&nbsp;</span>
-      <span>Questions</span>
-    </div>
-    <div>
-      <ReviewOverview />
-      <MostHelpfulReviews />
-      <SortAndProgress />
-      <IndividualReview />
-      <NextPageAndProgress />
-    </div>
-  </div>
-);
+import ReviewOverview from './ReviewOverview';
+import MostHelpfulReviews from './MostHelpfulReviews';
+import SortAndProgress from './SortAndProgress';
+import Reviews from './Reviews';
+import NextPageAndProgress from './NextPageAndProgress';
+/*--------------------------------*/
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      oneItem: {},
+    };
+  }
+
+  componentDidMount() {
+    this.getTodos();
+  }
+
+  getTodos() {
+    axios.get('http://localhost:3000/review/1')
+      .then((response) => {
+        this.setState({
```

Comment:
For the setState method, to follow airbnb style guide, we have to passing callback. I think you got problems msg from ESlint. So for your setState method can be written as following.

this.setState({() => ({oneItem: response.data})}, () => (console.log (this.state.oneItem)))

Console.log is just for checking that setState works or not.  

Let me know if the above setState doesn't work.